src/ResourceManager/KeyVault/Commands.KeyVault/Commands/RemoveAzureKeyVaultCertificate.cs

```diff
@@ -27,7 +27,7 @@ namespace Microsoft.Azure.Commands.KeyVault
         SupportsShouldProcess = true,
         ConfirmImpact = ConfirmImpact.High,
         HelpUri = Constants.KeyVaultHelpUri)]
-    [OutputType(typeof(KeyVaultCertificate))]
+    [OutputType(typeof(DeletedCertificateBundle))]
```

Comment:
Not sure what you mean.. The response of this cmdlet is instantiated on line 93:

DeletedCertificateBundle certBundle = null;

The DataServiceClient (invoked to delete the cert) returns:

 DeletedCertificateBundle DeleteCertificate(string vaultName, string certName);